plugins/destination/mssql/client/write.go

```diff
@@ -0,0 +1,88 @@
+package client
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/cloudquery/plugin-sdk/plugins/destination"
+	"github.com/cloudquery/plugin-sdk/schema"
+	"github.com/cloudquery/plugin-sdk/specs"
+)
+
+func (c *Client) WriteTableBatch(ctx context.Context, table *schema.Table, resources [][]any) error {
```

Comment:
This shouldn't be implemented. Destination plugin can't be both managed and unmanaged. For mysql you most prob want unmanaged as mentioned on slack because for managed plugin this means it will have to create more connections which mssql is not great at just like pg.

Reply:
Thanks for pointing this out!